src/main/java/Moss/TaskList.java

```diff
@@ -0,0 +1,151 @@
+package moss;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+/**
+ * The TaskList class handles processing and executing user commands related to task management.
+ */
+public class TaskList {
+
+    /**
+     * Processes and executes user commands related to task management.
+     *
+     * @param message The user command to be processed.
+     * @param things The list of tasks being managed.
+     * @param storage The storage object for saving tasks.
+     * @throws MossException If there's an issue with processing commands or managing tasks.
+     */
+    public static void command(String message, ArrayList<Task> things, Storage storage) throws MossException {
```

Comment:
Could use a verb for the name of this function since `command` by itself might be ambiguous. Perhaps `executeCommand`?